jenkins/development.Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Fedora 32 pbench developement image
+#
+# We maintain this image with all the necessary user-space setup required to
+# run the various unit tests and functional tests in a common development
+# environment.
+#
+# The image is published to https://quay.io/pbench/development and "tagged"
+# using the branch name in which it works: `master`, `b0.70`, `b0.71`, etc.
+#
+# NOTE WELL: This image has no entry point.
+#
+# It is intended to be built on top of with additional entry point for running
```

Comment:
This raises my passive voice hackles :-) Maybe something like "The intention is that an image with an entry point for running a test (or a suite of tests) is built on top of this" if you still want passive voice - or "You should build an image on top of this, with an entry point for running a test (or a suite of tests) in pbench".

Reply:
Thanks, fixed.